crates/storage/src/collections/bitvec/bits256.rs

```diff
@@ -118,7 +118,7 @@ impl<'a> IterMut<'a> {
 
     /// Returns access for the given bit index with extended but valid lifetimes.
     fn get<'b>(&'b mut self, index: u8) -> BitRefMut<'a> {
-        unsafe { BitRefMut::new(extend_lifetime(&mut self.bits), index) }
+        unsafe { BitRefMut::new(extend_lifetime(self.bits), index) }
```

Comment:
What was the clippy error here? This kinda seems correct to me.

Reply:
```
warning: this expression borrows a reference (`&mut collections::bitvec::bits256::Bits256`) that is immediately dereferenced by the compiler
   --> crates/storage/src/collections/bitvec/bits256.rs:121:49                 
    |                                                                          
121 |         unsafe { BitRefMut::new(extend_lifetime(&mut self.bits), index) } 
    |                                                 ^^^^^^^^^^^^^^ help: change this to: `self.bits`
    |                   
    = note: `#[warn(clippy::needless_borrow)]` on by default                                                                                                   
    = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#needless_borrow
```